.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 
 python:
-  - "2.7"
```

Comment:
Why did you remove the 2.x Python. I thought we can do both.

Reply:
Somehow, it is currently a bug I try to deal with: Enums are not directly available in python 2.7. Still, in my master branch [link](https://github.com/kperun/nestml/blob/PyNestML/.travis.yml), it works quite well, but here, we the exactly the same setup, it results in strange import errors. So yes, we will provide 2.7 support, but currently, the are some bugs.

Reply:
Ok, take a look on:
https://stackoverflow.com/questions/26828206/importerror-no-module-named-enum
https://stackoverflow.com/questions/39670197/ubuntu-16-04-python-2-7-importerror-no-module-named-enum

Reply:
Fixed, thanks for the information. I used 
```
pip install --upgrade pip enum34
```
as an additional pre-installation step and it seems to solve the problem. Still, on my master branch, it works quite well without this. I will have to take a look at this matter.
All test work now fine and python 2.7. has been integrated for testing.